src/altscore/borrower_central/model/borrower_fields.py

```
@@ -132,16 +133,18 @@ def __init__(self, altscore_client):
-    def find_by_key(self, key: str, borrower_id: str):
+    def find_by_key(self, key: str, borrower_id: str,
+                    include_tests: bool = True, test_only: bool = False):
         with httpx.Client(base_url=self.altscore_client._borrower_central_base_url) as client:
+            params = build_test_params({
+                "key": key,
+                "borrower-id": borrower_id,
+                "per-page": 1,
+                "page": 1
+            }, include_tests=include_tests, test_only=test_only)
             fields_found_req = client.get(
                 f"/v1/borrower-fields",
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)